docs/topics/installing-vs-code-extension.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension on Visual Studio Code
```

Comment:
I think "on" is not the right preposition. You might install an extension "in" a product. I think "for" would be a better choice.